pynblint/repository.py

```diff
@@ -38,6 +39,17 @@ def retrieve_notebooks(self):
                     nb = Notebook(Path(root) / Path(f), repository_path=self.path)
                     self.notebooks.append(nb)
 
+    def retrieve_dependencies_files(self):
+        dirs_ignore = [
+            '.ipynb_checkpoints'
+        ]
+        for root, dirs, files in os.walk(self.path):
+            # `dirs[:] = value` modifies dirs in-place
+            dirs[:] = [d for d in dirs if d not in dirs_ignore]
+            for f in files:
+                if (f == "requirements.txt" or f == "setup.py" or f == "environment.yml") and f not in self.dependencies_files:
+                    self.dependencies_files.append(f)
```

Comment:
Torno a consigliarti di considerare tutti i file di requirements analizzati da Pimentel. Mi pare che ce ne fosse almeno un altro (Pipenv?)

A questi dovresti aggiungere `environment.yml`, che invece Pimentel non aveva considerato.

Potresti organizzarli in una lista e fare un controllo del tipo:

```python
if f in env_files_list ...
```